frontend/src/summary.pug

```diff
@@ -42,20 +42,20 @@
         input(v-if="isSafariBrowser", type="text", placeholder="yyyy-mm-dd",
           v-bind:value="filterSinceDate", v-on:keyup.enter="updateTmpFilterSinceDate",
           onkeydown="formatInputDateOnKeyDown(event)", oninput="appendDashInputDate(event)", maxlength=10)
-        input(v-else, type="date", v-bind:value="filterSinceDate", v-on:input="updateTmpFilterSinceDate",
-          onkeydown="return false", v-bind:min="minDate", v-bind:max="filterUntilDate")
+        input(v-else, type="date", name="since", v-bind:value="filterSinceDate", v-on:input="updateTmpFilterSinceDate",
+          v-bind:min="minDate", v-bind:max="filterUntilDate")
         label since
       .mui-textfield.summary-picker__date
         input(v-if="isSafariBrowser", type="text", placeholder="yyyy-mm-dd",
           v-bind:value="filterUntilDate", v-on:keyup.enter="updateTmpFilterUntilDate",
           onkeydown="formatInputDateOnKeyDown(event)", oninput="appendDashInputDate(event)", maxlength=10)
-        input(v-else, type="date", v-bind:value="filterUntilDate", v-on:input="updateTmpFilterUntilDate",
-          onkeydown="return false", v-bind:min="filterSinceDate", v-bind:max="maxDate")
```

Comment:
removed this `onkeydown` as there are no other way to test the date picker without typing into it.